### PR TITLE
Fix reload feed button does nothing

### PR DESCRIPTION
**Pull request by Stypox**
#### What is it?
- [x] Bugfix (user facing)
- [ ] Feature (user facing)
- [ ] Codebase improvement (dev facing)
- [ ] Meta improvement to the project (dev facing)

#### Description of the changes in your PR
This was caused by view binding PR. initListeners was being called before the bindings were assigned, and therefore the click listener was never setup. @Isira-Seneviratne 

#### Due diligence
- [x] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Stypox** [comment]
@XiangRongLin could you review this? :-)

**Review by XiangRongLin — CHANGES_REQUESTED** [review]
>  initListeners was being called before the bindings were assigned, and therefore the click listener was never setup

Add that reasoning to the commit message

**Comment by XiangRongLin** [comment]
Unrelated, but is it intended that it's possible for `_feedbinding` to be null while accessing it? For example here
https://github.com/TeamNewPipe/NewPipe/blob/5e544e611f5675e6f4e93beeeb1be06269da5cac/app/src/main/java/org/schabi/newpipe/local/feed/FeedFragment.kt#L95

Also since you are already there, to me it looks like `_errorBinding` does not need that weird setup. It can just be a `lateinit var`. Because unlike `_feedbinding` it's not set to null in `onDestroyView`

**Comment by Stypox** [comment]
> Also since you are already there, to me it looks like _errorBinding does not need that weird setup. It can just be a lateinit var. Because unlike _feedbinding it's not set to null in onDestroyView

`_errorBinding` will be removed completely in #5148, so I'm not taking care of it here (though I was tempted to ;-))

> If not, all usages of _feedbinding should be replaced with feedbinding, to not silently break features, but breaking them visibly

I agree, I'm going to remove them

**Comment by Stypox** [comment]
Done @XiangRongLin

File: app/src/main/java/org/schabi/newpipe/local/feed/FeedFragment.kt [hunk 1]
```diff
@@ -83,13 +82,12 @@ class FeedFragment : BaseListFragment<FeedState, Unit>() {
     }
 
     override fun onViewCreated(rootView: View, savedInstanceState: Bundle?) {
-        super.onViewCreated(rootView, savedInstanceState)
         _feedBinding = FragmentFeedBinding.bind(rootView)
         _errorBinding = feedBinding.errorPanel
+        super.onViewCreated(rootView, savedInstanceState)
```
[the remark this answers is not in the document]
Comment (XiangRongLin) [reply app/src/main/java/org/schabi/newpipe/local/feed/FeedFragment.kt:81]: @Stypox ping

(your change was moving the line from line 84 to 87)

**Review by XiangRongLin — APPROVED** [review]

**Pull request merged**